### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by luisbevilacqua**
## Objective

Create a template for Issues to make contributing with new ideas to the project and reporting/tracking bugs easier.

## Notes

I would also like to suggest adding **Bug** and **Feature Request** tags to Issues, what do you think about it?
This file is inside the .github/ folder to avoid cluttering the root directory, following the Github documentation: https://github.com/blog/2111-issue-and-pull-request-templates

## Checklist

- [x] I have read the **[CONTRIBUTING]** document.
- [x] I have updated the documentation accordingly. (If needed)
- [ ] I have created a _Codelog changefile_ with the changes made to the branch.
- [ ] I have created a test proving that my feature/fix does what it intends to do.

[CONTRIBUTING]: https://github.com/codus/codelog/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<!--- Also, include any error messages displayed -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+<!--- or ideas how to implement the addition or change -->
+
+## Steps to Reproduce (for bugs)
+<!--- Provide an unambiguous set of steps to reproduce this bug. -->
+<!---  Include code to reproduce, if relevant -->
```
Comment (emilianoLeite) [inline .github/ISSUE_TEMPLATE.md]: Maybe we could add
> \

💯

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+1.
+2.
+3.
+4.
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* Codelog version used:
+* Ruby version:
+* Operating System and version:
```
[the remark this answers is not in the document]
Comment (luisbevilacqua) [reply .github/ISSUE_TEMPLATE.md]: In **Current Behavior** there is a  comment saying "Also, include any error messages displayed", do you think a new section with a title like **Stacktrace (For bugs)** would make it clear for the user?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (luisbevilacqua) [reply .github/ISSUE_TEMPLATE.md]: Good idea! :100:

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (luisbevilacqua) [reply .github/ISSUE_TEMPLATE.md]: Or maybe add a direct reference to the _stacktrace_ in this comment, something like `Also, include any error messages displayed and the full stacktrace`

[the remark this answers is not in the document]
Comment (FernandaThomazinho) [reply .github/ISSUE_TEMPLATE.md]: I like the option of adding a new section **Stacktrace (For bugs)**

**Comment by emilianoLeite** [comment]
Sounds good :+1:

Em 16 de fev de 2018 9:53 AM, Luís Fernando Bevilacqua <notifications@github.com> escreveu:

@luisbevilacqua commented on this pull request.

________________________________

In .github/ISSUE_TEMPLATE.md<https://github.com/codus/codelog/pull/32#discussion_r168737877>:

> +
+
+
+## Steps to Reproduce (for bugs)
+
+
+1.
+2.
+3.
+4.
+
+## Your Environment
+
+* Codelog version used:
+* Ruby version:
+* Operating System and version:


Or maybe add a direct reference to the stacktrace in this comment, something like Also, include any error messages displayed and the full stacktrace

**Pull request merged**